docs/arcadia-users-group/20230314-intro-to-python-2/lesson.md

```diff
@@ -0,0 +1,863 @@
+# Introduction to Python, Part 2
+
+Follow along in **Binder** by clicking the badge below:  
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Arcadia-Science/arcadia-computational-training/das/pytutorial?labpath=docs%2Farcadia-users-group%2F20230314-intro-to-python-2%2Flesson.ipynb)
```

Comment:
can you update this now to main branch so it will resolve from there? i know it will break rn if we click it but then it's the one we want to use for the actual lesson

Reply:
Updated!